sycl/include/sycl/handler.hpp

```diff
@@ -438,14 +438,22 @@ class __SYCL_EXPORT handler {
   ///        is null if no secondary queue is associated with the submission.
   /// \param CallerNeedsEvent indicates if the event resulting from this handler
   ///        is needed by the caller.
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
+  // TODO: This function is not used anymore, remove it in the next
+  // ABI-breaking window.
```

Comment:
I don't think this is true. SYCL unittests are only built in non-preview mode and they seem to be using this. All of them would need to be updated once this is truly removed during next ABI breaking period.